modules/hedera-app/src/main/java/com/hedera/node/app/impl/App.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.impl;
+
+import com.hedera.node.app.Service;
+import com.hedera.node.app.service.contract.SmartContractService;
+import java.util.ArrayList;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+
+public class App {
```

Comment:
In my PR (https://github.com/hashgraph/hedera-services/pull/4295) which is about to merged, I called this "Hedera".

Reply:
already did by other PR